packages/core/tests/unit/components/navi-visualizations/pie-chart-test.ts

```diff
@@ -0,0 +1,297 @@
+import { A } from '@ember/array';
+import { set } from '@ember/object';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import PieChart, { PieChartOptions } from 'navi-core/components/navi-visualizations/pie-chart';
+import { createGlimmerComponent } from 'navi-core/test-support';
+import NaviFactResponse from 'navi-data/models/navi-fact-response';
+import RequestFragment from 'navi-core/models/bard-request-v2/request';
+import { C3Row } from 'navi-core/chart-builders/base';
+
+let Request: RequestFragment;
+
+module('Unit | Component | pie chart', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function(this: TestContext) {
+    await this.owner.lookup('service:navi-metadata').loadMetadata();
+    Request = this.owner.lookup('service:store').createFragment('bard-request-v2/request', {
+      columns: [
+        {
+          cid: 'cid_age(field=desc)',
+          field: 'age',
+          parameters: { field: 'desc' },
+          type: 'dimension',
+          source: 'bardOne'
+        },
+        {
+          cid: 'cid_totalPageViews',
+          field: 'totalPageViews',
+          parameters: {},
+          type: 'metric',
+          source: 'bardOne'
+        },
+        {
+          cid: 'cid_uniqueIdentifier',
+          field: 'uniqueIdentifier',
+          parameters: {},
+          type: 'metric',
+          source: 'bardOne'
+        },
+        {
+          cid: 'cid_network.dateTime',
+          field: 'network.dateTime',
+          parameters: {
+            grain: 'day'
+          },
+          type: 'timeDimension',
+          source: 'bardOne'
+        }
+      ],
+      filters: [
+        {
+          field: 'network.dateTime',
+          parameters: {
+            grain: 'day'
+          },
+          operator: 'bet',
+          values: ['2015-12-14 00:00:00.000', '2015-12-15 00:00:00.000'],
+          type: 'timeDimension',
+          source: 'bardOne'
+        }
+      ],
+      table: 'tableA',
```

Comment:
Should align with the dateTime column
```suggestion
      table: 'network',
```